app/file.go

```diff
@@ -1082,11 +1084,30 @@ func (s *Server) getFileInfo(fileID string) (*model.FileInfo, *model.AppError) {
 }
 
 func (a *App) GetFileInfo(fileID string) (*model.FileInfo, *model.AppError) {
-	fileInfo, err := a.Srv().getFileInfo(fileID)
-	if err == nil {
+	fileInfo, appErr := a.Srv().getFileInfo(fileID)
+	if appErr != nil {
+		return nil, appErr
+	}
+
+	firstInaccessibleFileTime, appErr := a.isInaccessibleFile(fileInfo)
+	if appErr != nil {
+		return nil, appErr
+	}
+	if firstInaccessibleFileTime > 0 {
+		return nil, model.NewAppError("GetFileInfo", "app.file.cloud.get.app_error", nil, "", http.StatusForbidden)
+	}
```

Comment:
This is the only place we use `app.isInaccessibleFile`. Can we just return a bool instead of the `firstInaccessibleFileTime` and then checking if it's greater than 0? 

The name `isInaccessibleFile` also suggests a boolean return. Also, how about changing the name to `isAccessibleFile` so that we don't have the mental overhead of the negative (inverse).